crates/egui/src/context.rs

```diff
@@ -1240,6 +1240,9 @@ impl Context {
     ///
     /// This will repaint the current viewport.
     pub fn request_repaint(&self) {
+        if self.input(|i| i.viewport().minimized.unwrap_or(false)) {
+            return;
```

Comment:
I don't think this is the right solution.

The core problem is that when minimized, `App::update` is being called with a bad viewport size. We should either make sure `App::update` isn't called when minimized, or if called it is done so with the size of the window from when it was large.

This fix will just fix _some_ reasons for why `App::update` might be called, but not all. For instance, calling `request_repaint_after`, and then minimizing, will still call `App::update`.

TL;DR: the fix needs to into `eframe`, not `egui`

Reply:
@emilk
Dear emilk,

I'm not sure if this is the right solution, but I've been able to avoid the problem with this method after testing it for about a year.

I think App::update() should continue to work and should not only output to the screen.

In most cases, most users only call request_repaint(), and there are no cases where request_repaint_after() is called.

Please make a Pull Request for this temporarily until I find a better way. 
If there is a better way, please fix it.

Thank you, emilk.

Reply:
If you want, let it return only on Windows.

Reply:
Changed :
crates/eframe/src/native/run.rs 

The test results are normal.